### ISSUE TITLE
required_version warnings for module not installed from registry

### ISSUE THREAD

**Issue by levinotik** [terraform-linters/tflint-ruleset-aws#376]
I am using tflint via [pre-commit-terraform](https://github.com/antonbabenko/pre-commit-terraform) and running into an issue with some of the rules. I'm not sure if this is due to the way I'm running tflint or if this is truly a bug. 

The issue I'm running into is warnings about missing `required_version` for modules that are defined locally, not from any registry. According to the terraform docs:

> Version constraints are supported only for modules installed from a module registry

Yet, for each of my own local modules, tflint produces the following error:

```bash
Warning: terraform "required_version" attribute is required (terraform_required_version)

  on  line 0:
   (source code not available)

Reference: https://github.com/terraform-linters/tflint-ruleset-terraform/blob/v0.1.0/docs/rules/terraform_required_version.md

\e[0m\e[33mTFLint in vpc/:\e[0m
1 issue(s) found:
```

For example, each of the following produces this warning:

```hcl
module "vpc" {
  source = "./vpc"
  vpc_id = var.vpc.vpc_id
}

# Creating security groups
module "security_group" {
  source      = "./security_group"
  environment = var.environment
  vpc         = module.vpc.vpc
}

module "rds" {
  source          = "./rds"
  subnet_ids      = module.vpc.private_subnets
  security_groups = [module.security_group.sg_db.security_group_id]
  depends_on      = [
    module.security_group
  ]
}
module "sqs" {
  source      = "./sqs"
  environment = var.environment
}
# .... etc...
```

I may very well be missing something basic. Please let me know if I am. Thanks!
